src/WCSimTuningParameters.cc

```diff
@@ -11,9 +11,14 @@ WCSimTuningParameters::WCSimTuningParameters()
  // Default values
  rayff=0.75;
  bsrff=2.50;
- abwff=1.30;
+ abwff=1.30424242;
```

Comment:
Where does the difference come from?

Reply:
I do not know actually. Anyway, for all HK analyses, we used 1.30 in the mac file, so I have now changed it back to 1.30 as you suggest. All my thanks